datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -322,9 +334,10 @@ void shutdownProducer() {
       // quickly to avoid delaying/blocking the sender thread. Thus schedule the actual producer.close() on a separate
       // thread
       _producerCloseExecutorService.submit(() -> {
-        _log.info("KafkaProducerWrapper: Closing the Kafka Producer");
+        int timeout = fastClose ? FAST_CLOSE_TIMEOUT_MS : _producerCloseTimeoutMs;
+        _log.info("KafkaProducerWrapper: Closing the Kafka Producer timeout: {}", timeout);
```

Comment:
nit: Closing the Kafka Producer with timeout

Reply:
done